07-space.Rmd

```diff
@@ -328,6 +347,31 @@ tm_shape(roads_agg) + tm_fill("N. Crashes") +
   tm_shape(crashes_outside_roads) + tm_dots(col = "blue")
 ```
 
+The following code can be used to download roads data, filter it to show only main roads, and save as an .rds file for later use in your R project. In this example the code is downloading data for the county of Essex.
+
+```{r}
+library(geofabrik)
+remotes::install_github("itsleeds/geofabrik")
```

Comment:
This should now be:

```r
 remotes::install_github("osmextract/geofabrik") 
```

Reply:
Have tested in my existing script and it works. Not sure how I implement this my end as when I go to edit the chapter I can only see the version before I made my suggested edits,

Reply:
OK no worries. I made a typo, should have said

```
remotes::install_github("itsleeds/osmextrac")
```

Check it out here: https://itsleeds.github.io/osmextract/